Chapters/DynamicPresenter/DynamicPresenter.md

```diff
@@ -69,262 +68,236 @@ presenter layout remove: button2.
 
 ![Removing a button.](figures/layout5.png width=40&label=layout5)
 
-**Note.** What you see here is that all the changes happen simply by creating a new instance of a given layout and sending messages to it.  It means that programs can create complex logic of the dynamic behavior of a presenter.
+What you see here is that all the changes happen simply by creating a new instance of a given layout and sending messages to it.  It means that programs can define complex logic for the dynamic behavior of a presenter.
 
 ### Creating a presenter that dynamically adds buttons with random numbers
 
-We will create a presenter in which we will add and remove dynamically buttons. We will create a new class called `DynamicButtonsPresenter`.
+We will create a presenter in which we will add and remove buttons dynamically. We will create a new class called `DynamicButtons`.
 
-![A presenter that dynamically adds buttons](figures/layout8.png width=40&label=layout8)
+![A presenter that dynamically adds buttons.](figures/layout8.png width=40&label=layout8)
 
 ```
-SpPresenter << #DynamicButtonsPresenter
-	slots: { #addPresenterButton . #removePresenterButton . #textPresenter . #canRemovePresenter };
-	package: 'DynamicButtons'
+SpPresenter << #DynamicButtons
```

Comment:
Let us keep it but in general I always postfix Presenter to presenter class.

Reply:
> Let us keep it but in general I always postfix Presenter to presenter class.

I do not :-) I have removed some postfixes in other chapters as well. The names are shorter, and I believe that the postfix does not add value. It is similar to the collection classes. They are not called `ArrayCollection` and `BagCollection`. On the other hand, I understand that the postfix indicates the kind of classes a class belongs to.